browser/resources/settings/brave_sync_page/brave_sync_setup.js

```diff
@@ -82,13 +86,14 @@ Polymer({
   },
 
   submitSyncCode_: async function () {
+    console.log('submitSyncCode_ 000')
```

Comment:
do we need that log?

Reply:
no, this is forgotten leftover, useless in release. I removed that in https://github.com/brave/brave-core/pull/12300/commits/b99f8c1071ca66166ef91ec1b19e4cccede18dd0, thanks for finding.